brain_games/games/progression.py

```diff
@@ -0,0 +1,23 @@
+from random import choice
+from typing import Dict
+
+from brain_games.games.utils import generate_game_round_data, get_random_number
+
+RULES = "What number is missing in the progression?"
+
+
+def progression() -> Dict[str, str]:
+    """
+    Forms brain-progression game round data
+
+    :return: brain-progression game round data
+    :rtype: Dict[str, str]
+    """
+    step = get_random_number(1, 10)
+    start = get_random_number(0, 40)
+    end = start + step * get_random_number(5, 10)
```

Comment:
[optional] `start + step * get_random_number(5, 10)` will be easier to read if `get_random_number(5, 10)` was called smth like `progreesion_size`